sdk/communication/azure-communication-network-traversal/README.md

```diff
@@ -0,0 +1,82 @@
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/communication/azure-communication-identity) | [Package (Pypi)](https://pypi.org/project/azure-communication-identity/) | [API reference documentation](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/communication/azure-communication-identity) | [Product documentation](https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-python)
+
+
+# Getting started
+### Prerequisites
+- Python 2.7, or 3.6 or later is required to use this package.
+- You must have an [Azure subscription](https://azure.microsoft.com/free/)
+- A deployed Communication Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/powershell/module/az.communication/new-azcommunicationservice) to set it up.
+### Install the package
+Install the Azure Communication Identity client library for Python with [pip](https://pypi.org/project/pip/):
+Install the Azure Communication Relay Client library for Python with [pip](https://pypi.org/project/pip/):
+
+```bash
+pip install azure-communication-identity
+pip install azure-communication-network-traversal
+```
+
+# Key concepts
+
+### Initializing Relay Client
+```python
+# You can find your endpoint and access token from your resource in the Azure Portal
+import os
+from azure.communication.network.traversal import CommunicationRelayClient
+from azure.identity import DefaultAzureCredential
+from azure.communication.identity import CommunicationIdentityClient
+
+connection_str = "endpoint=ENDPOINT;accessKey=KEY"
+endpoint = "https://<RESOURCE_NAME>.communication.azure.com"
+
+# To use Azure Active Directory Authentication (DefaultAzureCredential) make sure to have
+# AZURE_TENANT_ID, AZURE_CLIENT_ID and AZURE_CLIENT_SECRET as env variables.
+# We also need Identity client to get a User Identifier
+identity_client = CommunicationIdentityClient(endpoint, DefaultAzureCredential())
+relay_client = CommunicationRelayClient(endpoint, DefaultAzureCredential())
+           
+
+#You can also authenticate using your connection string
+identity_client = CommunicationIdentityClient.from_connection_string(self.connection_string)
+relay_client = CommunicationRelayClient.from_connection_string(self.connection_string)
+
+```
+
+### Getting the configurationRelay 
+```python
+
+# We need a user from Identity
+user = identity_client.create_user()
+request = CommunicationRelayConfigurationRequest(id=user.properties.get('id'))
+
+relay_configuration = relay_client.get_relay_configuration(request)
+
+for iceServer in relay_configuration.ice_servers:
+    print(iceServer)
+```
+
+# Troubleshooting
+The Azure Communication Relay client will raise exceptions defined in [Azure Core][azure_core].
+
+# Next steps
+## More sample code
+
```

Comment:
Add a link to the sample directory here

Reply:
For JS the build checks failed when it looked for the link of the samples (since it doesn't exist yet because it hasn't been checked in). So it is very likely to happen the same here. I will add the link on a separate PR once the code is checked in.

Reply:
Fair